forge-gui/res/cardsfolder/o/of_one_mind.txt

```diff
@@ -1,7 +1,7 @@
 Name:Of One Mind
 ManaCost:2 U
 Types:Sorcery
-S:Mode$ ReduceCost | ValidCard$ Card.Self | Type$ Spell | Amount$ 2 | EffectZone$ All | IsPresent$ Creature.Human+YouCtrl | IsPresent2$ Creature.nonHuman+YouCtrl | Description$ CARDNAME costs {2} less to cast if you control a Human creature and a non-Human creature.
+S:Mode$ ReduceCost | ValidCard$ Card.Self | Type$ Spell | Amount$ 2 | EffectZone$ All | IsPresent$ Creature.Human+YouCtrl | CheckSVar$ Count$Valid Creature.nonHuman+YouCtrl | Description$ CARDNAME costs {2} less to cast if you control a Human creature and a non-Human creature.
```

Comment:
Why Change this?

Reply:
fixes the card for now because `StaticAbility.checkConditions` does not support IsPresent2
we could probably try and merge it into CardTraitBase checks but that is a bigger refactor